packages/interface/src/components/admin-menu-toggle/index.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { Button } from '@wordpress/components';
+import { useRef, useEffect, useState } from '@wordpress/element';
+import { ESCAPE, TAB } from '@wordpress/keycodes';
+import { __ } from '@wordpress/i18n';
+import { wordpress } from '@wordpress/icons';
+
+function AdminMenuToggle() {
+	const buttonRef = useRef();
+	const toggleMenu = useToggle( { ref: buttonRef } );
+
+	const { isActive, postType } = useSelect( ( select ) => {
+		const { getCurrentPostType } = select( 'core/editor' );
+		const { isFeatureActive } = select( 'core/edit-post' );
+		const { getPostType } = select( 'core' );
+
+		return {
+			isActive: isFeatureActive( 'fullscreenMode' ),
+			postType: getPostType( getCurrentPostType() ),
+		};
+	}, [] );
+
+	if ( ! isActive || ! postType ) {
+		return null;
+	}
+
+	return (
+		<Button
+			className="edit-post-fullscreen-admin-menu-toggle"
```

Comment:
This classname should probably be updated to reflect the new package.

Reply:
Nice catch. Updated.